dbms/src/Storages/DeltaMerge/StoragePool.cpp

```diff
@@ -375,18 +367,18 @@ PageStorageRunMode StoragePool::restore()
             data_storage_writer = std::make_shared<PageWriter>(run_mode, /*storage_v2_*/ nullptr, data_storage_v3);
             meta_storage_writer = std::make_shared<PageWriter>(run_mode, /*storage_v2_*/ nullptr, meta_storage_v3);
 
-            max_log_page_id = global_storage_pool->getLogMaxId(ns_id);
-            max_data_page_id = global_storage_pool->getDataMaxId(ns_id);
-            max_meta_page_id = global_storage_pool->getMetaMaxId(ns_id);
+            max_log_page_id = global_storage_pool->log_storage->getMaxId(ns_id);
```

Comment:
Use `log_pagestorage_v3` to replace `global_storage_pool->log_storage` pls.
also dito meta/data and below.

Reply:
Fixed, please check again